specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/examples/WorkItemConfigGet.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "api-version": "2015-05-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "my-resource-group",
+    "resourceName": "my-component",
+    "workItemConfigId" : "Visual Studio Team Services"
+  },
+  "responses": {
+    "200": {
+      "body": {"ConnectorId":"d334e2a4-6733-488e-8645-a9fdc1694f41","IsDefault":true,"ConfigDisplayName":"Visual Studio Team Services","Id":"Visual Studio Team Services","ConfigProperties":"{\"VSOAccountBaseUrl\":\"https://testtodelete.visualstudio.com\",\"ProjectCollection\":\"DefaultCollection\",\"Project\":\"todeletefirst\",\"ResourceId\":\"d0662b05-439a-4a1b-840b-33a7f8b42ebf\",\"ConfigurationType\":\"Standard\",\"WorkItemType\":\"Bug\",\"AreaPath\":\"todeletefirst\",\"AssignedTo\":\"\"}"}
```

Comment:
>"body": {"ConnectorId":"d334e [](start = 6, length = 29)

This single line is a bit hard to read. Can you format it to match the rest of the JSON. (Same comment on the other examples.)

Reply:
Will do.